src/pages/TodoPage/TodoPageContainer.tsx

```diff
@@ -5,9 +5,17 @@ import { ITodo } from '../../models/ITodo';
 
 export const TodoPageContainer: React.FC = () => {
   const [todos, setTodos] = useState([] as ITodo[]);
+  const [modalType, setModalType] = useState('');
   const [visibleModal, setVisibleModal] = useState(false);
+  const [visibleModalWithEditFeature, setVisibleEditModalWithEditFeature] =
```

Comment:
What's the difference between visibleModal and visibleModalWithEditFeature?

Reply:
emmm, literally the same logic, deleted